docs/editors/emacs.md

```diff
@@ -28,10 +28,17 @@ Next, update your Emacs configuration to load `lsp-scala` along with its
 dependencies
 
 ```el
+;; Add melpa-stable to your packages repositories
+(add-to-list 'package-archives '("melpa-stable" . "http://stable.melpa.org/packages/") t)
+
+;; Enable defer and ensure by default for use-package
+(setq use-package-always-defer t
+      use-package-always-ensure t)
+
 ;; Enable scala-mode and sbt-mode
 (use-package scala-mode
-  :interpreter
-  ("scala" . scala-mode))
+  :mode "\\.s\\(cala\\|bt\\)$")
```

Comment:
I've never had to do anything special for this to work.

Reply:
This enables `scala-mode` in `.sbt` and `.scala` files. Reading the doc for [`:mode` and `:interpreter`](https://jwiegley.github.io/use-package/keywords/#mode-interpreter) it seems to do the same. I can revert that one.

Reply:
I don't have `:mode` or `:interpreter` in mine, and I get scala-mode for both `*.sbt` and `*.scala`.  What you've done is explicit and doesn't hurt anything, but I got this for free.

Reply:
Good to know :+1: